Content.Server/Chemistry/EntitySystems/ReagentDispenserSystem.cs

```diff
@@ -49,6 +51,7 @@ public override void Initialize()
             SubscribeLocalEvent<ReagentDispenserComponent, ReagentDispenserDispenseReagentMessage>(OnDispenseReagentMessage);
             SubscribeLocalEvent<ReagentDispenserComponent, ReagentDispenserEjectContainerMessage>(OnEjectReagentMessage);
             SubscribeLocalEvent<ReagentDispenserComponent, ReagentDispenserClearContainerSolutionMessage>(OnClearContainerSolutionMessage);
+            SubscribeLocalEvent<ReagentDispenserComponent, ReagentDispenserInsertStorageMessage>(OnInsertStorageMessage);
```

Comment:
Can't you handle this in shared and then use PredictedSendMessage?

Not too big of a deal though because I'm planning on making a lot of reagent dispensers and chemmaster predicted very soon (like will probably start today) anyway.

Reply:
there's not a corresponding shared system for the reagent dispenser; i would prefer to defer that for a proper move-to-shared effort

Reply:
Ah, you're so right. I saw the SharedReagentDispenser file and just assumed it surely had _something_ predicted. Nope :P